system/include/webgpu/README.md

```diff
@@ -13,27 +13,28 @@ The C++ files are **currently** Dawn-specific, but included in Emscripten for
 better compatibility with Dawn: it has roughly the same API as Dawn's copy, but
 is included here because it is strongly tied to an exact `webgpu.h` revision.
 
-Dawn additionally autogenerates two "snippets" that are used in Emscripten:
-- `library_webgpu_enum_tables.js`, which is pasted into [`library_webgpu.js`](../../../src/library_webgpu.js)
-- `webgpu_struct_info.json`, which is pasted into [`struct_info.json`](../../../src/struct_info.json).
-
-The file `webgpu_cpp_chained_struct.h` is also needed from Dawn's source,
-though it changes infrequently.
-
-Once that's done, you need to update the auto-generated files with the commands below:
-
-```
-./tools/maint/gen_struct_info.py
-./tools/maint/gen_struct_info.py --wasm64
-./tools/maint/gen_sig_info.py
-```
+To update these bindings from Dawn:
```

Comment:
We may want to use an ordered list 1,2,3,4 instead of bullet points as they need to be in ordered, especially for the last part where we auto-generate files right?